### DISCUSSION TITLE
Begin transcribing at audio track position.

### DISCUSSION THREAD

**Discussion by StarMariner** [openai/whisper#1905] [Ideas]
whisper command in linux , I don't see any option to start transcribing audio at a specific position.

Does anyone think this may be a useful feature?

I have been transcribing audio to srt text, it works well. The problem is, I sometimes have to cancel whisper mid transcription. I could write a script to fix this, but that is not very useful outside of the whisper command. If an option is possible, I'd like to see it universal for all audio formats that whisper can handle.  srt uses comma to separate seconds from milli seconds others use a period . 
`whisper thingy.mp3 --audio-start 00:06:51 --output_format srt`

**Comment by ryanheise** [comment]
There is a PR to do this here: #1838

In your case, you would use `whisper thingy.mp3 --clip_timestamps 411 --output_format srt`
